source/modules/collection-api/sale/overview.md.erb

```diff
@@ -1,19 +1,20 @@
 ## Sale Activity
 
-The following functions provide Inspetor with information on sales that happen within your platform. The main activities that we track are the following:
+You can provide sale-related activity via the REST API. The `/sales` endpoint supports `POST` and `PUT` operations with the following arguments:
 
-- <a href="#sale-creation">trackSaleCreation</a>
-- <a href="#sale-updates">trackSaleUpdate</a>
 
-All functions for tracking Sales activity require a [InspetorSale](#inspetorsale) object as an argument and return `true` upon successful execution. Otherwise, one of the following exceptions will be thrown:
-
-Exception | Description
---------- | -----------
-**[SaleException](#inspetorsaleexception)**         | The Transfer object provided is not a valid one
-**[ItemException](#inspetoritemexception)**         | The Item object provided on the Sale is not a valid one
-**[PaymentException](#pinspetoraymentexception)**   | The Payment object provided on the Sale is not a valid one
-**[generalException](#inspetorgeneralexception)** | The timestamps that you passed on the objects are not valid ones
-**[TrackerException](#trackerexception)**   | An internal error occurred *Hopefully this never happen*
+Property       | When is it required?   | Type   | Description
+--------       | ---------------------  | ----   | -----------
+sale_id        | `POST`, `PUT`          | String | The unique identifier for the sale within your platform
+account_id     | `POST`                 | String | The id of the user who is making the purchase
+status         | `POST`,                | String | The status of the sale. *The allowed values are: `accepted`, `declined`, `pending`, `refunded`, and `manual_analysis`*
```

Comment:
I guess it should be `aprroved` instead of `accepted`

Reply:
Yup, semantically I believe that's more correct (damn, Lourenço correcting _my_ English?!?)

Reply:
ahahahaha the student becomes the teacher

Reply:
If we're gonna change it here, we also gotta change it on the transfer schema.